tests/bot/exts/backend/sync/test_cog.py

```diff
@@ -60,23 +60,6 @@ def response_error(status: int) -> ResponseCodeError:
 class SyncCogTests(SyncCogTestCase):
     """Tests for the Sync cog."""
 
-    @mock.patch("botcore.utils.scheduling.create_task")
-    @mock.patch.object(Sync, "sync_guild", new_callable=mock.MagicMock)
-    def test_sync_cog_init(self, sync_guild, create_task):
-        """Should instantiate syncers and run a sync for the guild."""
-        # Reset because a Sync cog was already instantiated in setUp.
-        self.RoleSyncer.reset_mock()
-        self.UserSyncer.reset_mock()
-
-        mock_sync_guild_coro = mock.MagicMock()
-        sync_guild.return_value = mock_sync_guild_coro
-
-        Sync(self.bot)
-
-        sync_guild.assert_called_once_with()
-        create_task.assert_called_once()
-        self.assertEqual(create_task.call_args.args[0], mock_sync_guild_coro)
-
```

Comment:
This should likely be kept actually, since it's testing that the syncers are getting instantiated on cog load, so still a useful test.

Reply:
Yeah, I suppose it should be kept.

Reply:
87edeff7347f011c2317cd4b3681bea5bbe07185